controllers/cluster_scripts/cloud_init_control_plane.yaml

```diff
@@ -31,6 +31,16 @@ write_files:
       storageProfile: "%s"
       filesystem: "%s"
     ---
+- path: /etc/environment
```

Comment:
We can skip this file and directly put the `%s` in the `/root/control_plane.sh` below, correct?

Reply:
Done